.github/workflows/create-prerelease-branch.yml

```
@@ -109,7 +109,7 @@ jobs:
       - name: Hop into Groom branch
         id: groombranch
         run: |
-          git checkout -b groom_changelog_test_${{ steps.version.outputs.version }}
+          git checkout -b groom_changelog_${{ steps.version.outputs.version }}
```

can we use the name naming pattern for all the branches, prerelease is prerelease-{stable|beta}/{version}
```suggestion
          git checkout -b groom-changelog-${{github.event.inputs.bump_type}}/${{ steps.version.outputs.version }}
```

---

+1 to what @JamesBurnside said.

---

will do!